typescript/apitesters/customerInfo.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  LogInResult,
+  CustomerInfo,
+  PurchasesEntitlementInfo,
+  PurchasesEntitlementInfos,
+  PurchasesStoreTransaction
+} from "../src";
+
+function checkLoginResult(result: LogInResult) {
```

Comment:
About the API tests. I was thinking that, while we can have them here, we should also keep them in the hybrids just in case.